decidim-admin/app/views/decidim/admin/components/edit.html.erb

```diff
@@ -5,25 +5,7 @@
   </h2>
 </div>
 
-<div class="item__edit">
-  <div class="item__edit-menu">
-    <div class="item__edit-menu-sticky">
-      <%= decidim_form_for([current_participatory_space, @form], url: components_path(type: params[:type]), method: "post", html: { class: "form form-defaults new_component" }) do |form| %>
-        <p class="item__edit-menu-title">
-          <%= t("sections", scope: "decidim.participatory_processes.admin.participatory_processes.form") %>
-        </p>
-        <%= link_to t(".title", name: t("#{manifest.name}.name", scope: "decidim.components")), new_component_path(type: manifest.name, anchor: "title", name: t("#{manifest.name}.name", scope: "decidim.components")), class: "item__edit-menu-anchor" %>
-        <% if form.object.settings? %>
-          <%= link_to t("global_settings", scope: "decidim.admin.components.form"), new_component_path(type: manifest.name, anchor: "global_settings"), class: "item__edit-menu-anchor" %>
-        <% end %>
-        <% if current_participatory_space.has_steps? %>
-          <%= link_to t("step_settings", scope: "decidim.admin.components.form"), new_component_path(type: manifest.name, anchor: "step_settings"), class: "item__edit-menu-anchor" %>
-        <% elsif form.object.default_step_settings? %>
-          <%= link_to t("default_step_settings", scope: "decidim.admin.components.form"), new_component_path(type: manifest.name, anchor: "default_step_settings"), class: "item__edit-menu-anchor" %>
-        <% end %>
-      <% end %>
-    </div>
-  </div>
+<div class="item__edit item__edit-1col">
```

Comment:
Is this on https://nightly.decidim.org/admin/participatory_processes/angel-drill/components/16/edit, right?  What's this menu doing, is it even visible?

Reply:
This is the same menu highlighted in #11650 , that has been fixed by : https://github.com/decidim/decidim/pull/11681

Reply:
Ah ok, then it makes sense why I couldn't find it in nightly 🤣